api/src/main/java/org/pmiops/workbench/shibboleth/ShibbolethConfig.java

```diff
@@ -0,0 +1,29 @@
+package org.pmiops.workbench.shibboleth;
+
+import java.util.concurrent.TimeUnit;
+import org.pmiops.workbench.auth.UserAuthentication;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.shibboleth.api.ShibbolethApi;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.web.context.annotation.RequestScope;
+
+@org.springframework.context.annotation.Configuration
+public class ShibbolethConfig {
+  public static final String X_APP_ID_HEADER = "X-App-ID";
+
+  @Bean
+  @RequestScope(proxyMode = ScopedProxyMode.DEFAULT)
+  public ShibbolethApi shibbolethApi(
+      UserAuthentication userAuthentication, WorkbenchConfig workbenchConfig) {
+    ApiClient apiClient = new ApiClient();
+    apiClient.setBasePath(workbenchConfig.firecloud.shibbolethApiBaseUrl);
+    apiClient.addDefaultHeader(X_APP_ID_HEADER, workbenchConfig.firecloud.xAppIdValue);
+    apiClient.setDebugging(workbenchConfig.firecloud.debugEndpoints);
```

Comment:
I opted to use the standard "firecloud" API client config variables here. Even though Shibboleth is a distinct service, I felt it was simpler to use this standard set of API client variables rather than either (1) proliferate with add'l Shibboleth-specific configs or (2) use the ApiClient defaults. I'm open to objections here.

Reply:
This is probably fine